setup.py

```diff
@@ -18,6 +18,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url='https://github.com/alteryx/evalml/',
+    python_requires='>=3.8, <4',
```

Comment:
Thoughts on this? This is what pip looks at when determining if your python version would work with a library. Saw this i[n the WW repo](https://github.com/alteryx/woodwork/commit/19681ae1190bb7b027d62513137167e61a31b905#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R32).

Reply:
Looks good to me